editor/sidebar/featured-image/index.js

```diff
@@ -67,17 +74,17 @@ const applyConnect = connect(
 	( state ) => {
 		return {
 			featuredImageId: getEditedPostAttribute( state, 'featured_media' ),
+			isOpened: isEditorSidebarPanelOpened( state, PANEL_NAME ),
```

Comment:
Wondering if we should bind props using `PANEL_NAME` not only in `mapStateToProps`, but also in `mapDispatchToProps`, so toggle callback in the visual component doesn't need to know/pass the value.